client/app/layout/header/header.component.js

```diff
@@ -1,9 +1,18 @@
+function backgroundSelector () {
+	if (window.location.pathname == "/"){
```

Comment:
Go ahead and inject $state and use state.name.  Also use the triple eq for this please.